packages/@okta/vuepress-site/docs/guides/oin-universal-logout-overview/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Build Universal Logout for your app
+meta:
+  - name: description
+    content: Provides an overview of Universal Logout and how to build a Universal Logout endpoint
+---
+
+## Universal Logout
+
+When an Identity Provider (IdP) like Okta detects identity threats or responds to employee termination events, it can prevent the user from signing in to apps in the future. However, this doesn't affect a user's existing sessions or tokens within an app.
+
+Universal Logout enables an IdP, or a security incident management tool, to indicate to an app that it should revoke the user's existing sessions and log the user out.
+
+A Universal Logout endpoint must be built for the app to handle logout requests. This endpoint receives a request to log a user out and then attempts to revoke all sessions and tokens for the user. The endpoint then returns a result indicating success or failure.
+
+**[Read the full API Specification](https://okta-universal-logout.redoc.ly/openapi/logout/logout/tag/UniversalLogout/)**
+
+If you're building an app that's used by enterprise customers, and would like to empower your customers to instantly mitigate risks across their ecosystem, read on for how you can support Universal Logout with Okta.
+
+## Revocation details
+
+When the Universal Logout endpoint receives a request, it's a clear communication that the user's existing sessions and tokens should be revoked as fast as possible.
+
+Often, it's not possible to instantaneously revoke all sessions and tokens. When you use JWT access tokens or when you deploy an app across multiple independent data centers, instantaneous revocation doesn't occur. Because of this, a "Success" response to the logout request indicates that the app is doing everything within its power to log out all credentials.
+
+In particular, the app should revoke:
+
+* All OAuth 2.0 refresh tokens for the user
+* All session cookies for the user
+
+The app isn't expected to revoke access tokens, as it may not be technically possible. If your app can revoke access tokens, then go ahead and do it. However, revoking access tokens isn't required to consider the logout flow successful.
+
+## Universal Logout endpoint
+
+The actual endpoint URL is up to the discretion of the app developer building the endpoint for the app.
+
+### Endpoint authentication
+
+The request to the Universal Logout endpoint requires authentication so that your app knows the request is coming from Okta. When the app launches, Okta should be able to support your existing API authentication scheme, especially if it's one of the following:
```

Comment:
"When the app launches" sounds weird, I think because it's not clear whether "launch" refers to publication of the app or invocation of the app. Can we just drop that part?